app/API/version2/questions/models.py

```diff
@@ -105,19 +110,40 @@ def upvote_question(self, question_id):
         return make_response(jsonify({
                 "status": 200,
                 "msg": "Question",
-                "data": result
+                "data":self.quiz_json(result)
                 }))
 
     def downvote_question(self, question_id):
-        question = Helper.check_if_question_posted_exists(self, question_id)
+        question = Helper.check_ques(self,question_id)
+        if not question:
+            return {"message": "That question does not exist"}, 404
+        user_id = question[3]
+        votes = question[6]
+        if user_id:
+            if votes > 0:
+                return {"Message": "you can't vote again"}
```

Comment:
Should this response have a Status code?

Reply:
Worked on all the feedback. Thanks for the feedback